dist/images/ovndb-raft-functions.sh

```diff
@@ -29,9 +29,9 @@ ready_to_join_cluster() {
   if [[ $? != 0 ]]; then
     return 1
   fi
-  target=$(ovn-${db}ctl --db=${transport}:${init_ip}:${port} ${ovndb_ctl_ssl_opts} --data=bare --no-headings \
+  target=$(ovn-${db}ctl --db=${transport}:[${init_ip}]:${port} ${ovndb_ctl_ssl_opts} --data=bare --no-headings \
```

Comment:
This works with IPv4? I don't think it's _documented_ as being allowed...

Reply:
This does work with IPv4. I thought it was cleaner this way since it works, but if you prefer, I can add the [ ] for only IPv6. There are several places in `dist/images/ovndb-raft-functions.sh` and `dist/images/ovnkube.sh` I do this. `${init_ip}` is easy to change because all of it's uses will require [ ] for IPv6. There are a few variables in `dist/images/ovnkube.sh` that don't always need the [ ] for IPv6, so it may get a bit messier (variable with [ ] and variable without [ ], or command line repeated for different modes).

Reply:
I guess since this is just internal code, not something we're shipping, it's fine like this; if OVN changes in the future and stops supporting brackets for IPv4 addresses, we can just fix it then.